charts/app-config/templates/draft-content-store-proxy-service.yaml

```
@@ -0,0 +1,10 @@
+{{ if .Values.enableDraftContentStoreProxy }}
+# route all requests for draft-content-store via draft-content-store-proxy
+apiVersion: v1
+kind: Service
+metadata:
+  name: draft-content-store
+spec:
+  type: ExternalName
+  externalName: draft-content-store-proxy
```

This needs to be the fully-qualified name unfortunately (but that's ok because this doesn't need to work with any other namespace).
```suggestion
  externalName: draft-content-store-proxy.apps.svc.cluster.local
```